pyproject.toml

```diff
@@ -15,6 +15,7 @@ dependencies = [
   "torchvision ~= 0.14.1",
   "pytorch-lightning ~= 1.6.5",
   "torchmetrics == 0.6.0",
+  "numpy == 1.23.5", # https://github.com/pytorch/pytorch/issues/91516
```

Comment:
It looks like the newer PyTorch 2.0 has resolved the issue: https://github.com/pytorch/pytorch/pull/91517#issuecomment-1477998981

Could you add a note in the comment? I think we will upgrade to PyTorch 2.0 soon after `torchmetrics` fixes mAP.